docs/installation-guide-mac.rst

```diff
@@ -0,0 +1,62 @@
+:orphan:
+
+Installation guide for macOS
+============================
+
+Follow these instructions to install the CityEnergyAnalyst (CEA) on a macOS system (tested with Mojave 10.14.5) from the source
+
+Prerequisites
+~~~~~~~~~~~~~
+
+* Download and install Git (64-bit).
+* Download and install Github Desktop (64-bit).
+* Download and install Miniconda(64-bit) for Python 2.7.
+* Download and install CMake.
+* Install DAYSIM version >= 4.0: Open Terminal and run the following commands
+
+  - ``git clone https://github.com/MITSustainableDesignLab/Daysim.git``
+  - ``mkdir build && cd build && cmake -DBUILD_HEADLESS=on -DCMAKE_INSTALL_PREFIX=$HOME/Daysim ../Daysim && make && make install``

```

Comment:
I get errors in this step...

Reply:
`mkdir build && cd build && cmake -DBUILD_HEADLESS=on -DCMAKE_INSTALL_PREFIX=$HOME/Daysim ../Daysim && make && make install`
This step appears to have worked for me...? 🤷‍♀